Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -545,6 +545,8 @@ - (void)viewWillAppear:(BOOL)animated
     // or on another device.
     [self hideInputIfNeeded];
 
+    [self setNavigationTitle];
```

Comment:
We're also calling this 14 lines down. If we really need to call it twice, can you leave an explanatory comment for this precarious situation?